deliaioanac/assignment1/KthToLast.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <stdlib.h>
+
+using namespace std;
```

Comment:
I am sorry but am not sure what you are referring to, I can't find the comment. However, I might have an idea what it's about. I know that, as a general rule, in production things like "using namespace" should be avoided, as is real-life code it would be, due to the many intricate namespaces used, fairly ambiguous. I'll remove it, hoping this is what you were talking about :)